llama_stack/providers/tests/inference/test_model_registration.py

```diff
@@ -16,6 +16,14 @@
 
 
 class TestModelRegistration:
+    def provider_supports_custom_names(self, provider) -> bool:
+        """Returns True if the provider supports custom model names."""
```

Comment:
nit: I think these comments are not needed because the method name itself is quite self explanatory